src/main/java/uk/gov/hmcts/reform/unspec/model/CaseData.java

```diff
@@ -126,5 +126,6 @@ public boolean hasNoOngoingBusinessProcess() {
     private final LocalDateTime applicant1ResponseDeadline;
     private final LocalDateTime applicant1ResponseDate;
     private final LocalDateTime takenOfflineDate;
+    private final LocalDateTime takenOfflineByStaffDate;
```

Comment:
This is new case field `takenOfflineByStaffDate` introduced, I don't see this set any where in the code.